pybossa/cloud_store_api/s3.py

```diff
@@ -195,8 +195,10 @@ def get_content_and_key_from_s3(s3_bucket, path, conn_name=DEFAULT_CONN,
             secret = app.config.get('FILE_ENCRYPTION_KEY')
         cipher = AESWithGCM(secret)
         content = cipher.decrypt(content)
-    if type(content) == bytes:
+    try:
         content = content.decode()
+    except (UnicodeDecodeError, AttributeError):
+        pass
```

Comment:
We expect to pass decoding when an exception occur. this could happen to any byte contents which is not a byte string. Hence, seems logging not required.

Reply:
Added logging to test and confirm it covers all cases. Upon testing we can remove logging that is not required. Thanks!